experta/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '1.9.4'
+__version__ = '1.9.7'
```

Comment:
is this expected?

Reply:
Yep, I already increased the version twice before for other changes, the current, last one is for python >3.6 compatibility